src/components/GalleryLightBox/GalleryLightBox.tsx

```diff
@@ -117,107 +107,111 @@ export const GalleryLightBox = ({
           {slides.map((slide: GalleryLightBoxSlideProps, index: number) => {
             const srcSet = `${slide.mediaSources.gallery_full} 769w, ${slide.mediaSources.gallery_full_hi} 1538w`;
             const infoPaneContent = slide.title
-              ? `<h2 className="cc-gallery-lightbox__info-title">${slide.title}</h2>${slide.caption}`
+              ? `<h2 class="cc-gallery-lightbox__info-title">${slide.title}</h2>${slide.caption}`
               : slide.caption;
 
             return (
               <Slide
```

Comment:
The vast majority of changes here are structural to present a more flat HTML structure to give greater styling flexibility across breakpoints